core/example/solvers/pme.h

```diff
@@ -0,0 +1,54 @@
+#ifndef TDS_EWALD_INCLUDED
+#define TDS_EWALD_INCLUDED
+
+#include <Cabana_AoSoA.hpp>
+#include <Cabana_MemberTypes.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <iostream>
+#include "definitions.h"
+
+class TPME
+{
+  public:
+    // constructor with accuracy
+    TPME(double,ParticleList,ParticleList,double,double,double);
```

Comment:
Can you add names to the variables in header file functions?

Reply:
Good call - I added some description and names.

Reply:
I guess I meant more:
```
TPME( double accuracy_threshold, ParticleList particles, ParticleList mesh, double x_width, double y_width, double z_width );
```